routeros/resource_ip_dhcp_client_option_test.go

```diff
@@ -0,0 +1,47 @@
+package routeros
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+const testIpDhcpClientOptionAddress = "routeros_ip_dhcp_client_option.test_dhcp"
+
+func TestAccIpDhcpClientOptionTest_basic(t *testing.T) {
+	for _, name := range testNames {
+		t.Run(name, func(t *testing.T) {
+			resource.Test(t, resource.TestCase{
+				PreCheck: func() {
+					testAccPreCheck(t)
+					testSetTransportEnv(t, name)
+				},
+				ProviderFactories: testAccProviderFactories,
+				CheckDestroy:      testCheckResourceDestroy("/ip/dhcp-client-option", "routeros_ip_dhcp_client_option"),
```

Comment:
Please fix the resource path: /ip/dhcp-client/option

Reply:
oh sorry did not see the notification ! thanks